js/modules/k6/ws/ws.go

```diff
@@ -188,8 +189,12 @@ func (*WS) Connect(ctx context.Context, url string, args ...goja.Value) (*WSHTTP
 
 	defer func() { _ = conn.Close() }()
 
-	tags["status"] = strconv.Itoa(httpResponse.StatusCode)
-	tags["subprotocol"] = httpResponse.Header.Get("Sec-WebSocket-Protocol")
+	if state.Options.SystemTags["status"] {
+		tags["status"] = strconv.Itoa(httpResponse.StatusCode)
+	}
+	if state.Options.SystemTags["subproto"] {
+		tags["subproto"] = httpResponse.Header.Get("Sec-WebSocket-Protocol")
```

Comment:
Don't remember if I'm the one that made this change :), but if we change `subprotocol` to `subproto` I suspect the cloud service for example needs to be changed as well (@ppcano?).

Reply:
I'm the one who did it :smile:  I suggested it in an earlier comment in this PR and when nobody protested I assumed it was OK... :grinning: My rationale was that since we call the HTTP protocol tag `proto`, the subprotocol tag should be `subproto` for consistency. I can easily change it back if necessary.

Reply:
@na-- @robingustafsson 

> I suspect the cloud service for example needs to be changed as well 

Nothing has to be changed in the cloud service. The `subprotocol` tag is only used on the filter search and the tag filter works on any existing `tags` of the test results.

Reply:
@na-- 😄no worries, the name change makes sense 
@ppcano Ok, thought we had some special handling to show Websocket metrics in the breakdown?

Reply:
@robingustafsson 

To double check, I have reviewed the code, and we don't use the `subprotocol` tag for anything. The `ws` package emits different metrics (`ws_connecting`, `ws_session_duration` ....) and we check the metric name to handle it appropriately.

https://docs.k6.io/docs/k6-websocket-api

Reply:
@ppcano Ah, ok good.
@na-- I think the PR is good to merge then.

Reply:
Great, I'll merge it after I update the docs